policyengine_canada/variables/gov/provinces/nu/tax/income/credits/nu_cost_of_living_credit.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_canada.model_api import *
+
+
+class nu_cost_of_living_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Nunavut cost of living credit"
+    unit = CAD
+    definition_period = YEAR
+    defined_for = ProvinceCode.NU
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.nu.tax.income.credits.cost_of_living
+        income = person("individual_net_income", period)
+        armed_deduction = person(
+            "canadian_armed_forces_and_personnel_deduction", period
+        )
+        adjusted_income = max_(0, income - armed_deduction)
```

Comment:
Make this its own PolicyEngine variable since it's used in both places and computed separately in the [form](https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5014-tc/5014-tc-22e.pdf), e.g. `nu_cost_of_living_credit_adjusted_net_income`